xbmc/dialogs/GUIDialogMediaFilter.cpp

```diff
@@ -474,7 +474,9 @@ void CGUIDialogMediaFilter::InitializeSettings()
 
       if (filter.settingType == SettingType::Integer)
       {
-        int min, interval, max;
+        int min = 0;
```

Comment:
no complaints, just a remark https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#54-initialization => `int min{};`
would be in line with the arrays then, but honestly my personal preference is your version